cmd/healthz/main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/gravitational/trace"
+
+	pb "github.com/gravitational/satellite/agent/proto/agentpb"
+	"github.com/gravitational/satellite/healthz/checks"
+	"github.com/gravitational/satellite/healthz/config"
+	"github.com/gravitational/satellite/healthz/handlers"
+	"github.com/gravitational/satellite/healthz/utils"
+)
+
+func main() {
+	if err := run(); err != nil {
+		log.Fatal(err)
+		fmt.Printf("ERROR: %v\n", err.Error())
+		os.Exit(255)
+	}
+}
+
+func run() error {
+	cfg := config.Config{}
+	config.ParseCLIFlags(&cfg)
+
+	if cfg.Debug {
+		trace.EnableDebug()
+		log.SetLevel(log.DebugLevel)
+	}
+	log.SetFormatter(&log.TextFormatter{})
+	log.SetOutput(os.Stderr)
+
+	log.Debug(trace.Errorf("starting using config: %#v", cfg))
```

Comment:
I'm using `trace.Errorf` because it's easy way to prepend message with line position `[source/file.go:1234]`. To make it possible using logrus alone I need to write some hooks or custom loggers.